README.md

```diff
@@ -60,6 +60,7 @@ WAGTAILMARKDOWN = {
     "extensions": [],  # optional. a list of python-markdown supported extensions
     "extension_configs": {},  # optional. a dictionary with the extension name as key, and its configuration as value
     "extensions_settings_mode": "extend",  # optional. Possible values: "extend" or "override". Defaults to "extend".
+    "tab_length": 4,  # optional. Sets the tab_length used by python-markdown. Defaults to 4.
```

Comment:
I am a bit confused. You state

> The behavior of using two-space indent for nesting can be confusing to some users

but then mention that the default for this optional config param is 4.

```suggestion
    "tab_length": 4,  # optional. Sets the length of tabs used by python-markdown to render the out. This is the number of spaces used to replace with a tab character. Defaults to 4.
```

Reply:
Ah, thanks. I meant to say "four-space" in the PR description.